chassis/configuration/configuration/master/invoker/example-external-resources-invoker.conf.xml

```diff
@@ -131,6 +131,7 @@
 
     <bean id="googlePageNew" class="com.griddynamics.jagger.invoker.http.HttpInvokerConfiguration">
         <property name="query" ref="googlePageQuery"/>
+        <property name="endpoint" value="http://www.google.com/search"/>
     </bean>
```

Comment:
This poped-up after I added "context:annotation-config"  - which activates Spring annotations like Required. So, in code it is marked as required but here it is not set as supposed to.